### PR TITLE
Us615 shutter app

### PR DESCRIPTION
**Pull request by naikajah**
Initial merge for CDN and Shutter page

### CODE REVIEW

File: environments/sbox/shutter.tfvars [hunk 1]
```diff
@@ -0,0 +1,12 @@
+shutter_storage     = "default"
+env                 = "sbox"
+cdn_sku             = "Standard_Microsoft"
+resource_group_name = "shutter-app-rg"
+
+# Applications associated with default storage account shutter static website.
+# Teams that need a default shutter page should add there app frontend names to the list below.
```
Comment (timja) [inline environments/sbox/shutter.tfvars]:
```suggestion
# Teams that need a default shutter page should add their app frontend names to the list below.
```

File: modules/shutter_page/cdn_endpoints.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
+resource "azurerm_cdn_endpoint" "shutter_endpoint" {
+  count                     = length(var.shutter_apps)
+  name                      = "hmtcs-${split(".", var.shutter_apps[count.index])[0]}-shutter"
+  profile_name              = azurerm_cdn_profile.main.name
+  location                  = "West US"
```
Comment (timja) [inline modules/shutter_page/cdn_endpoints.tf]: why West US?

File: modules/shutter_page/custom_domains.tf [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# data "template_file" "customdomain" {
+#   template = "${file("${path.cwd}/templates/custom_domain.json")}"
+# }
+
+# resource "azurerm_template_deployment" "custom_domain" {
```
Comment (timja) [inline modules/shutter_page/custom_domains.tf]: this isn't working?

File: modules/shutter_page/templates/custom_endpoint.json [hunk 4]
```diff
@@ -0,0 +1,99 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "profile_name": {
+            "defaultValue": "hmcts-shutter-page-nonprod",
```
Comment (timja) [inline modules/shutter_page/templates/custom_endpoint.json]: possibly remove this? defaults can be a pain as sometimes they accidentally get used.

File: modules/shutter_page/templates/custom_endpoint.json [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    "variables": {
+        "endpoint": "[concat(parameters('profile_name'), '/', parameters('endpoint_name'))]",
+        "storageBlob": "[replace(parameters('originStorageBlob'),'.', '-')]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Cdn/profiles/endpoints",
+            "apiVersion": "2019-04-15",
+            "name": "[variables('endpoint')]",
+            "location": "WestUs",
```
Comment (timja) [inline modules/shutter_page/templates/custom_endpoint.json]: same q here, does it only work in west us?

File: modules/shutter_page/cdn_endpoints.tf (hunk 2 above)
Comment (naikajah) [reply modules/shutter_page/cdn_endpoints.tf]: CDN profile is global and in order to work with global, it needs to have WEST US in the location.

File: modules/shutter_page/custom_domains.tf (hunk 3 above)
Comment (naikajah) [reply modules/shutter_page/custom_domains.tf]: It was working earlier but last night I saw some errors 

> "Sorry, it looks like there was an error on our end. Please contact Support if you keep having this problem."

So I have commented this bit and will enable this but once I have a reply from MS.

File: modules/shutter_page/templates/custom_endpoint.json (hunk 4 above)
Comment (naikajah) [reply modules/shutter_page/templates/custom_endpoint.json]: Yes, I will remove this bit.

File: modules/shutter_page/templates/custom_endpoint.json (hunk 5 above)
Comment (naikajah) [reply modules/shutter_page/templates/custom_endpoint.json]: answered above

**Pull request merged**